public/app/features/dashboard-scene/scene/layouts-shared/paste.ts

```diff
@@ -41,6 +41,21 @@ export interface PanelStore {
   gridItem: GridLayoutItemKind | AutoGridLayoutItemKind;
 }
 
+export type ClipboardType = 'panel' | 'row' | 'tab';
+
+export function whatIsInClipboard(): ClipboardType | null {
+  if (store.exists(LS_PANEL_COPY_KEY)) {
```

Comment:
We could probably refactor this to only use a single key and differentiate based on the `kind` property of the object